integration_tests/packages.yml

```diff
@@ -2,12 +2,8 @@ packages:
   - package: fivetran/ad_reporting
```

Comment:
It looks like there are some upgrades we should apply to ensure we are testing the latest versions of all the packages.

<img width="1088" alt="image" src="https://github.com/fivetran/dbt_fivetran_utils/assets/74217849/43be7ea6-ab9c-4347-8ca6-78fc2e47a857">

Reply:
Updated!